ga4gh/ga2vcf.py

```diff
@@ -0,0 +1,345 @@
+"""
+Conversion utility that fetches data from the GA4GH server and
+writes to VCF files
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+
+import pysam
+
+import ga4gh.ga2utils as ga2utils
+import ga4gh.request_factory as requestFactory
+
+
+class VcfException(Exception):
+    pass
+
+
+class VcfMetadataInfoLine(object):
+
```

Comment:
Deep in the pysam code it calls a numbertype attribute, which is why this needs data needs to be wrapped in an object and __getattr__ overridden 

https://github.com/pysam-developers/pysam/blob/50698a6e2b8c39dc0d59172554a4e902681014b0/pysam/cvcf.pyx#L427